src/components/Icons/Error.js

```diff
@@ -118,22 +119,20 @@ const errorIcons = {
 };
 
 const ErrorIcon = ({ type, size, color, children, ...props }) =>
```

Comment:
I think you should remove `color` as an explicitly named prop from the destructured props, so devs can pass it through if they need to override the theme color.